### ISSUE TITLE
Incorporate OCR as part of derivative creation process

### ISSUE THREAD

**Issue by dinahhandel** [sul-dlss/common-accessioning#229]
Currently, the only content that can be OCRed must go through Goobi. Ideally, OCR would take place during the derivative creation process in assembly, so that content accessioned through preassembly, the self-deposit app, and Goobi could all take advantage of OCR technology. This would also allow us to explore other OCR software, and have more control over the OCR software with regards to monitoring.

**Comment by andrewjbtw** [comment]
Is this being closed in favor of another issue? It remains a feature request.

**Comment by mjgiarlo** [comment]
@andrewjbtw no. I believe ZenHub did a weird last night when @ndushay was doing some pruning. @jcoyne and I re-opened dozens of other tickets. If you see any tickets closed by @ndushay yesterday around 6:30 PT... just re-open 'em.
